run-dljc.sh

```diff
@@ -100,179 +100,98 @@ fi
 
 ### Script
 
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
 # clone DLJC if it's not present
 if [ ! -d do-like-javac ]; then
     git clone https://github.com/kelloggm/do-like-javac
 fi
```

Comment:
Add an else clause that does `git pull`, to ensure that the scripts are up to date.